### DISCUSSION TITLE
Min Max Close buttons coming on left side for arabic language is selected for the system

### DISCUSSION THREAD

**Discussion by Sunilam** [microsoft/WindowsAppSDK#3443] [General]
HI Team,

Its a Winui3 application.

how to keep title bar min/max/close buttons to right side when arabic language is selected.

![image](https://user-images.githubusercontent.com/7436855/220016948-d5abefdc-e5a7-4251-bc6b-633fc5b2279c.png)

Now i have selected arabic as default language in my system and launched my application. Here Min/Max/Close icons in the title bar is showing in the left side.

We want those min/max/close buttons to be shown in the right side of the title bar always for all the languages.

Note: Min/Max/Close buttons shows on right side of the title bar normally when english language is selected in the system.

**Answer from MSDN: https://learn.microsoft.com/en-us/answers/questions/1167352/how-to-keep-title-bar-min-max-close-buttons-to-rig

when can we expect this feature of showing min/max/close button to top right corner when arabic language is selected in the system.**

Best Regards,
Sunil

**Comment by Sunilam** [comment]
Hi Team,
Can we have answer for the above question.

**Reply by ghost1372** [reply]
If you have only rtl language (i mean only one language not multilanguage) you can only set flowdirection of root page
And you dont need to change application language in this way. Also i think you can achive this with titlebar customization if you find any solution please let me know i need this too.

**Reply by Sunilam** [reply]
If you have only rtl language (i mean only one language not multilanguage) you can only set flowdirection of root page
And you dont need to change application language in this way. --- CAn you please tell me how to achive this?

**Reply by Gitii** [reply]
@Sunilam have you tried this?
https://stackoverflow.com/questions/24007761/how-to-change-layout-of-form-title-bar-to-rtl

**Comment by peterjtan** [comment]
The AppWindow's title bar renders right-to-left (RTL) if it detects that the window style `WS_EX_LAYOUTRTL` is set. You could try removing `WS_EX_LAYOUTRTL` from the AppWindow's window style by calling `SetWindowLong`, and that should satisfy your requirement.
